test.js

```diff
@@ -533,11 +533,11 @@ describe('RS256 JWT token validation', function () {
       headers: { Authorization: `Bearer ${tokens.rs256MissingKey}` }
     })
 
-    expect(response.statusCode).toEqual(500)
+    expect(response.statusCode).toEqual(401)
     expect(JSON.parse(response.body)).toEqual({
```

Comment:
you may want to consider changing these assertions to extract the JSON-encoded response body with response.json() instead of parsing the raw body